packages/gatsby-plugin-netlify/src/create-redirects.js

```diff
@@ -0,0 +1,16 @@
+import fs from "fs"
+import pify from "pify"
+
+const writeFile = pify(fs.writeFile)
+
+export default function writeRedirectsFile(pluginData, redirects) {
+  const { publicFolder } = pluginData
+
+  // https://www.netlify.com/docs/redirects/
+  const data = redirects.map(redirect => {
+    const status = redirect.isPermanent ? 301 : 302
+    return `${redirect.fromPath}  ${redirect.toPath}  ${status}`
+  })
+
+  return writeFile(publicFolder(`_redirects`), data.join(`\n`))
+}
```

Comment:
Maybe we could add a plugin option to disable this as well?

Reply:
Is there any harm in writing an empty `_redirects` file? Or are you concerned that we might override an existing one? (Guess it would be worth checking to see if we should append vs write.)

Reply:
Right—someone might want to make one by hand through the static folder rather than go through generation in this plugin.

Reply:
I'll create a follow-up PR.